actionkit/rest.py

```diff
@@ -1,7 +1,7 @@
 import requests
 import json
 
-from urlparse import urlparse
+from urllib import parse
```

Comment:
it doesn't look like parse or urlparse is used at all in this file, so we can safely delete this import